README.md

```diff
@@ -41,6 +41,11 @@
     pip install -e ../arches
     ```
 
+1. Create a settings_local.py file with `DEBUG=True`:
```

Comment:
Would it be worth it to specify that this should be run in `arches-lingo/arches_lingo`?